### ISSUE TITLE
My redis traces are not being connected to my span

### ISSUE THREAD

**Issue by mjgorman** [opentracing-contrib/python-redis#10]
I'm doing a little sample POC of OpenTracing for our team and I am getting the traces for redis, but they're separate from my spans which are running those redis commands. I'm setting a tag and that all works, but in jaeger they just come out as separate traces making it sort of useless.

![screen shot 2018-12-07 at 3 14 30 pm](https://user-images.githubusercontent.com/5741377/49677286-d691fa80-fa32-11e8-90e6-0fae6dda5fe5.png)

<img width="1066" alt="screen shot 2018-12-07 at 3 10 23 pm" src="https://user-images.githubusercontent.com/5741377/49677289-deea3580-fa32-11e8-86ec-4a89b4726928.png">

**Comment by carlosalberto** [comment]
Hey @mjgorman 

If I understand correctly, you want the `INCR` and `GET` to automatically become children of your `redis-span` Span, correct? If that's the case, this is part of an ongoing effort, as part of adopting the OT 2.0 API (which adds active `Span` propagation & handling): https://github.com/opentracing-contrib/python-redis/tree/v1.0.0

The ETA for this effort to be finished is 1-2 weeks ;)

**Comment by mjgorman** [comment]
@carlosalberto great to hear. Glad it wasn't just something I was doing incorrectly.

**Comment by carlosalberto** [comment]
Hey @mjgorman 

I just released the `1.0.0` version, which should fix this. Simply remember to update your code to look like this:

```python
with tracer.start_active_span('redis-span') as scope:
    # Here your Redis commands and other related code.
    # As 'redis_span' is the active Span, all new Spans
    # under this block will become implicit children of it.
```

Let me know ;)

**Comment by mjgorman** [comment]
Switching start_span to start_active_span throw an error. 
```
  File "02b.py", line 36, in do_redis_things
    with tracer.start_active_span('redis-span') as scope:
  File "/Users/michael.gorman/sites/opentracing/python/venv/lib/python3.7/site-packages/opentracing/tracer.py", line 126, in start_active_span
    return self._noop_scope
AttributeError:  'Tracer' object has no attribute '_noop_scope'
```

**Comment by carlosalberto** [comment]
Hey @mjgorman 

Thanks for the answer. So you are using Jaeger, right? It seems they still haven't released a version that implements the OT 2.0 api (which automatically detects the parent/child relationship).

As far as I know, LightStep, DataDog and Instana clients support this already ;) (I can poke the Jaeger guys and maybe help them to have this released, too). Let me know.

**Comment by mjgorman** [comment]
@carlosalberto Ah I see. Thank you for the informative response. I'll take it up with them.
